File: jput-core/src/main/java/cz/encircled/jput/unit/UnitPerformanceResult.java
package cz.encircled.jput.unit;

/**
 * @author Vlad on 28-May-17.
 */
public class UnitPerformanceResult {

    public boolean isAverageLimitMet = true;

    public boolean isMaxLimitMet = true;

    public long runAverageTime;

    public long runMaxTime;

    public boolean isError() {
        return !isAverageLimitMet || !isMaxLimitMet;
    }

}

File: jput-core/src/main/java/cz/encircled/jput/unit/UnitPerformanceResult.kt
package cz.encircled.jput.unit

/**
 * @author Vlad on 28-May-17.
 */
data class UnitPerformanceResult(var isAverageLimitMet: Boolean = true, var isMaxLimitMet: Boolean = true,
                                 var runAverageTime: Long = 0, var runMaxTime: Long = 0) {

    val isError: Boolean
        get() = !isAverageLimitMet || !isMaxLimitMet

}
